src/pouchDB/types.ts

```diff
@@ -0,0 +1,120 @@
+export enum DesignDoc {
+  CountOfVotesForAnIdea = 'votes/idea-votes',
+  CountOfCommentsForAnIdea = 'comments/idea-comments',
+  CountOfLikesForAComment = 'likes/comment-likes',
+  CountOfTagsUsed = 'tags/tags-count',
+  HomePageFilter = 'filters/homepage',
+  IdeaDetailPageFilter = 'filters/ideaDetailPage',
+}
+
+export enum IndexDoc {
+  SortedByTypeVoteAuthor = 'type-votes-author-index',
+  SortedByTypeIdAuthor = 'type-id-author-index',
+  SortedByTypeIdeaidVotes = 'type-ideaId-votes-index',
+}
+
+interface Updatation {
```

Comment:
Shouldn't this be `Updation`? Unless `Updatation` is something else.
```suggestion
interface Updation {
```